### ISSUE TITLE
Make process flowchart sticky on page

### ISSUE THREAD

**Issue by sarahelizabeth** [sc3/crime-punishment#115]
....so that it stays in place while scrolling through the table of contents at larger viewpoints ONLY (no tablet)

**Comment by sarahelizabeth** [comment]
I found this but I'm having trouble implementing it @bepetersn do i need to include the whole "jQuery-Stickem" folder or just the "jquery.stickem.js" file?

**Comment by bepetersn** [comment]
Can @wilbertom or @eads try to do this, I don't see how to do it. Everytime my element sticks, it becomes taller and wider inexplicably. Affix doesn't even have a clear way to make the fixed element unfix after a certain point, which is important for this use case. Maybe we have to use javascript to get that behavior, instead of pure css. Another thing people seem to suggest is fixing a parent element and controlling the child element inside of it; maybe that will fix the weird size changes.

Either way, I don't want to fix this right now.

**Comment by eads** [comment]
As I told sarah this morning, please stop trying to make affix work here
and just let it scroll.

On Wed, Mar 26, 2014 at 10:13 AM, Brian Everett Peterson <
notifications@github.com> wrote:

> Can @wilbertom https://github.com/wilbertom or @eadshttps://github.com/eadstry to do this, I don't see how to do it. Everytime my element sticks, it
> becomes taller and wider inexplicably. Affix doesn't even have a clear way
> to make the fixed element unfix after a certain point, which is important
> for this use case. Maybe we have to use javascript to get that behavior.
> Another thing people seem to suggest is fixing a parent element and
> controlling the child element inside of it; maybe that will fix the weird
> size changes.
> 
> Either way, I don't want to fix this right now.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/sc3/chicago-justice/issues/115#issuecomment-38695773
> .

## 

David Eads - 773.354.2285
News applications developer, Chicago Tribune (
http://www.chicagotribune.com/news/data/)
Founder, FreeGeek Chicago (http://freegeekchicago.org)

"Medical statistics will be our standard of measurement: we will weigh life
for life and see where the dead lie thicker, among the workers or among the
privileged." -- Rudolf Virchow

**Comment by bepetersn** [comment]
Can you help me understand what "let it scroll" means? What Sarah said of
your commentary was TO use Bootstrap's affix instead of the stick'em jquery
plugin.

**Comment by eads** [comment]
Right. But you're also struggling with that. There's a bunch of tricks
you'll need to make it work that neither of you are going to acquire in the
next day, so just don't try and make it affix. I can show you those tricks
when the project is done.

Take it from a guy who likes sticky elements waaaay too much.

On Wed, Mar 26, 2014 at 10:28 AM, Brian Everett Peterson <
notifications@github.com> wrote:

> Can you help me understand what "let it scroll" means? What Sarah said of
> your commentary was TO use Bootstrap's affix instead of the stick'em jquery
> plugin.
> 
> On Mar 26, 2014 10:25 AM, "David Eads" notifications@github.com wrote:
> 
> > As I told sarah this morning, please stop trying to make affix work here
> > and just let it scroll.
> > 
> > On Wed, Mar 26, 2014 at 10:13 AM, Brian Everett Peterson <
> > notifications@github.com> wrote:
> > 
> > > Can @wilbertom https://github.com/wilbertom or @eads<
> > > https://github.com/eads>try to do this, I don't see how to do it.
> > 
> > Everytime my element sticks, it
> > 
> > > becomes taller and wider inexplicably. Affix doesn't even have a clear
> > > way
> > > to make the fixed element unfix after a certain point, which is
> > > important
> > > for this use case. Maybe we have to use javascript to get that
> > > behavior.
> > > Another thing people seem to suggest is fixing a parent element and
> > > controlling the child element inside of it; maybe that will fix the
> > > weird
> > > size changes.
> > > 
> > > Either way, I don't want to fix this right now.
> > > 
> > > ## 
> > > 
> > > Reply to this email directly or view it on GitHub<
> > > https://github.com/sc3/chicago-justice/issues/115#issuecomment-38695773>
> > > 
> > > .
> > 
> > ## 
> > 
> > David Eads - 773.354.2285
> > News applications developer, Chicago Tribune (
> > http://www.chicagotribune.com/news/data/)
> > Founder, FreeGeek Chicago (http://freegeekchicago.org)
> > 
> > "Medical statistics will be our standard of measurement: we will weigh
> > life
> > for life and see where the dead lie thicker, among the workers or among
> > the
> > privileged." -- Rudolf Virchow
> > 
> > ## 
> > 
> > Reply to this email directly or view it on GitHub<
> > https://github.com/sc3/chicago-justice/issues/115#issuecomment-38697377>
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/sc3/chicago-justice/issues/115#issuecomment-38697744
> .

## 

David Eads - 773.354.2285
News applications developer, Chicago Tribune (
http://www.chicagotribune.com/news/data/)
Founder, FreeGeek Chicago (http://freegeekchicago.org)

"Medical statistics will be our standard of measurement: we will weigh life
for life and see where the dead lie thicker, among the workers or among the
privileged." -- Rudolf Virchow

**Comment by eads** [comment]
By let it scroll I just mean don't try to apply position: fixed to the
element, just leave it as a statically positioned element and let it scroll
with the rest of the content.

**Comment by bepetersn** [comment]
So what you're trying to tell me is to to relax and not worry about it? :D

**Comment by eads** [comment]
Yes.

**Comment by bepetersn** [comment]
Can do! Think I'll take a nap.
